ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/mapper/defendant/DefendantMapper.java

```diff
@@ -45,74 +46,84 @@ public DefendantMapper(
         this.reDeterminationMapper = reDeterminationMapper;
     }
 
-    public CCDCollectionElement<CCDDefendant> to(TheirDetails theirDetails, Claim claim) {
+    public CCDCollectionElement<CCDRespondent> to(TheirDetails theirDetails, Claim claim) {
         requireNonNull(theirDetails, "theirDetails must not be null");
         requireNonNull(claim, "claim must not be null");
 
-        CCDDefendant.CCDDefendantBuilder builder = CCDDefendant.builder();
-        builder.responseDeadline(claim.getResponseDeadline());
-        builder.letterHolderId(claim.getLetterHolderId());
-        builder.defendantId(claim.getDefendantId());
-        builder.partyEmail(claim.getDefendantEmail());
-        builder.responseMoreTimeNeededOption(CCDYesNoOption.valueOf(claim.isMoreTimeRequested()));
-        builder.directionsQuestionnaireDeadline(claim.getDirectionsQuestionnaireDeadline());
-        builder.countyCourtJudgmentRequest(countyCourtJudgmentMapper.to(claim));
+        CCDRespondent.CCDRespondentBuilder respondentBuilder = CCDRespondent.builder();
+        respondentBuilder.responseDeadline(claim.getResponseDeadline());
+        respondentBuilder.letterHolderId(claim.getLetterHolderId());
+        respondentBuilder.defendantId(claim.getDefendantId());
+
+        CCDParty.CCDPartyBuilder partyDetail = CCDParty.builder();
+        partyDetail.emailAddress(claim.getDefendantEmail());
+
+        respondentBuilder.responseMoreTimeNeededOption(CCDYesNoOption.valueOf(claim.isMoreTimeRequested()));
+        respondentBuilder.directionsQuestionnaireDeadline(claim.getDirectionsQuestionnaireDeadline());
+        respondentBuilder.countyCourtJudgmentRequest(countyCourtJudgmentMapper.to(claim));
 
         claim.getSettlement().ifPresent(settlement ->
-            builder.settlementPartyStatements(
+            respondentBuilder.settlementPartyStatements(
                 settlementMapper.toCCDPartyStatements(settlement)
             )
         );
-        builder.settlementReachedAt(claim.getSettlementReachedAt());
+        respondentBuilder.settlementReachedAt(claim.getSettlementReachedAt());
 
-        claim.getResponse().ifPresent(toResponse(claim, builder));
-        theirDetailsMapper.to(builder, theirDetails);
+        claim.getResponse().ifPresent(toResponse(claim, respondentBuilder, partyDetail));
+        if (!claim.getResponse().isPresent()) {
```

Comment:
We are having toResponse method call as consumer functional interface. If we are going to have if else, it would need refactor as well, thats why I left as this.